borrowing/models.py

```diff
@@ -1,3 +1,17 @@
+from django.contrib.auth import get_user_model
 from django.db import models
 
-# Create your models here.
+from book.models import Book
+
+User = get_user_model()
+
+
+class Borrowing(models.Model):
+    borrow_date = models.DateField(auto_now_add=True)
+    expected_return_date = models.DateField()
+    actual_return_date = models.DateField(null=True, blank=True)
+    book = models.ForeignKey(Book, on_delete=models.CASCADE)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)

```

Comment:
Change to on_delete=models.PROTECT, add related_name="borrowings" for both fields